src/_includes/head.html

```diff
@@ -34,7 +34,9 @@
 <link href="{{ assets['favicon.ico'].digest_path }}" rel="icon" type="image/png">
 <link rel="canonical" href="{{ page.url | site.url | replace:'index.html',''}}">
 
+{% if jekyll.environment != "development" %}
```

Comment:
Let's use `{% if jekyll.environment == "production" %}` instead, otherwise you'll get exceptions from test runs.

Reply:
Nevermind, I just looked through the code and we're only using `production`/`development`.